src/components/DayPickerRangeController.jsx

```diff
@@ -1174,6 +1176,7 @@ export default class DayPickerRangeController extends React.PureComponent {
       transitionDuration,
       verticalBorderSpacing,
       horizontalMonthPadding,
+      forceRender
```

Comment:
this prop isn’t forcing this component to rerender, so it seems confusing to only pass it down.